kura/org.eclipse.kura.ai.triton.server/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,39 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: org.eclipse.kura.ai.triton.server
+Bundle-SymbolicName: org.eclipse.kura.ai.triton.server;singleton:=true
+Bundle-Version: 1.0.0.qualifier
+Bundle-Vendor: Eclipse Kura
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Service-Component: OSGI-INF/*.xml
+Bundle-ClassPath: .,
+ lib/com.google.protobuf_3.19.3.jar,
+ lib/grpc-netty-shaded-1.33.1.jar,
+ lib/grpc-protobuf-1.33.1.jar,
+ lib/grpc-stub-1.33.1.jar,
+ lib/grpc-context-1.33.1.jar,
+ lib/grpc-core-1.33.1.jar,
+ lib/grpc-protobuf-lite-1.33.1.jar,
+ lib/perfmark-api-0.19.0.jar,
+ lib/grpc-api-1.33.1.jar
+Bundle-ActivationPolicy: lazy
+Import-Package: com.google.common.base;version="25.0.0",
+ com.google.common.io;version="25.0.0",
+ com.google.common.util.concurrent;version="25.0.0",
+ com.google.gson;version="2.7.0",
+ javax.annotation;version="1.3.5",
+ org.apache.commons.io;version="2.4.0",
+ org.eclipse.kura;version="[1.0,2.0)",
+ org.eclipse.kura.ai.inference;version="[1.0,2.0)",
```

Comment:
If your bundle implements an interface marked as `ProvideType` then the package that provides that interface should be inported with a range like `[current_major.current_minor, current_major.current_minor+1)`

Reply:
Replaced inference api import with `org.eclipse.kura.ai.inference;version="[1.0,1.1)"`